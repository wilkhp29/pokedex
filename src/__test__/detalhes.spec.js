import React from 'react';
import { shallow } from 'enzyme';
import Detalhes from '../Views/detalhe';

import { Provider } from "react-redux";
import configureStore from "redux-mock-store";

const mockStore = configureStore();

const initalState = {
    pokedox:[],
    pokemon:{busca:'',pokemon:{},loading:false,pokemonBusca:{}}
}

const store = mockStore(initalState);

it('renderizando a Detalhes', () => {
    const wrapper = shallow(
    <Provider store={store}>
        <Detalhes />
    </Provider>
        ,{context:{store}});

    expect(wrapper).toMatchSnapshot();
});