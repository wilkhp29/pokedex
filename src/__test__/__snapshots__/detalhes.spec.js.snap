// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderizando a Detalhes 1`] = `ShallowWrapper {}`;
